src/v2/view-builder/views/phone/EnrollAuthenticatorPhoneView.js

```diff
@@ -1,9 +1,148 @@
+import { loc } from 'okta';
+import BaseView from '../../internals/BaseView';
 import BaseForm from '../../internals/BaseForm';
 import BaseFactorView from '../shared/BaseFactorView';
+import CountryUtil from '../../../../util/CountryUtil';
 
 const Body = BaseForm.extend({
+
+  className: 'phone-authenticator-enroll',
+
+  title () {
+    return loc('oie.phone.enroll.title', 'login');
+  },
+
+  subtitle () {
+    return loc('oie.phone.enroll.subtitle', 'login');
+  },
+
+  _changeView ({ changed }) {
```

Comment:
- more explicit like `handleMethodTypeChange`. 
- nit: doesn't have to prefix `_` to indicate it's private.